Sources/Stagehand/AnimatableProperty/AnimatableProperty+CoreGraphics.swift

```diff
@@ -56,76 +56,3 @@ extension CGRect: AnimatableProperty {
     }
 
 }
-
-extension CGAffineTransform: AnimatableProperty {
-
-    /// Interpolates between the `initialValue` and `finalValue`.
-    ///
-    /// This supports transforms that are composed of translations, scales, and rotations; where `M' = R * S * T * M`.
-    /// In order words, the matrix must be mutated in order of (1) translations, (2) scales, then (3) rotations. It does
-    /// not support transforms that have had a skew/distort applied.
-    public static func value(
-        between initialValue: CGAffineTransform,
-        and finalValue: CGAffineTransform,
-        at progress: Double
-    ) -> CGAffineTransform {
-        let initialRotation = initialValue.rotation
-
-        // Pick the shortest route to the between the transforms by adjusting the final angle by ±2π.
-        let calculatedFinalRotation = finalValue.rotation
-        let finalRotationCandidates = [
-            calculatedFinalRotation - 2 * .pi,
-            calculatedFinalRotation,
-            calculatedFinalRotation + 2 * .pi
-        ]
-        let finalRotation = finalRotationCandidates.min(by: { abs($0 - initialRotation) < abs($1 - initialRotation) })!
```

Comment:
I realized this actual regresses the rotation handling a bit, since it no longer finds the shortest rotation across 2π. Should be an easy fix though.

Reply:
Fixed this, so it now finds the shortest rotation delta between transforms. Unfortunately this had a significant affect on the performance... but still better than it was before this PR! It's now ~24% faster than the old method.

I have some ideas for how we could cache the decomposed values so we avoid some of the math each time. Not sure it's necessary yet, but might be a good future enhancement.